app/src/main/java/secapstone/helper/Artisan.java

```diff
@@ -21,6 +24,8 @@
     private String pinterest;
     private String instagram;
     private boolean customOrders;*/
+    private float moneyOwedFromCommunityLeader;
+    private List<Listing> listings = new ArrayList<Listing>();
```

Comment:
make "listings" into a hashmap with its product id as the key and the whole Listing as the value